Flat/config/GeneralTree.cfg

```diff
@@ -25,7 +25,41 @@ sf_lepID                   float
 sf_lepIso                  float
 sf_lepTrack                float
 sf_pho                     float
-# btag SFs - these are done differently now
+# btag SFs
+sf_cmvaweight            float
```

Comment:
Why are these booked as individual floats?

Reply:
did not edit the config with the enum changes